connection.go

```diff
@@ -891,6 +900,14 @@ func (c *Connection) close(fields ...LogField) error {
 // Close starts a graceful Close which will first reject incoming calls, reject outgoing calls
 // before finally marking the connection state as closed.
 func (c *Connection) Close() error {
+	// Remaining frames in sendCh must be drained and released,
```

Comment:
do we still want this, or is it a left-over from the previous iteration?

The PR description mentions only changes in `writeFrames`, and your other comment says "as discussed, prefer to keep all code that consume sendCh close together", which seems to contradict this piece of code?